base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -1582,9 +1582,16 @@ fn handle_reorg<T: BlockchainBackend>(
         .height -
         1;
     let removed_blocks = reorganize_chain(db, block_validator, fork_height, reorg_chain)?;
-    if removed_blocks.is_empty() {
-        Ok(BlockAddResult::Ok)
-    } else {
+    let num_removed_blocks = removed_blocks.len();
+    let num_added_blocks = added_blocks.len();
+
+    if reorg_required(num_removed_blocks, num_added_blocks) {
+        trace!(
```

Comment:
I think just add this to the `info!`. The debug could probably even be removed

Reply:
Thanks - will fix